<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeriodicTable</class>
 <widget class="QMainWindow" name="PeriodicTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Elements</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover{
        background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #ffd9aa,
                stop :   0.5 #ffbb6e, stop :   0.55 #feae42, stop :   1.0 #fedb74);
}

QPushButton {
        border: 1px solid #7d8392;
        border-radius: 2px;
        padding: 5px 15px 2px 5px;
        background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #f5f9ff,
                stop :   0.5 #c7dfff, stop :   0.55 #afd2ff, stop :   1.0 #c0dbff);
        color: #7d8392;
        font: bold large &quot;Arial&quot;;
        height: 30px;
}




QPushButton:pressed {
        background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #c0dbff,
        stop :   0.5 #cfd26f, stop :   0.55 #c7df6f, stop :   1.0 #f5f9ff);
        padding-top: 2px;
        padding-left: 3px;

}


QPushButton:on {
        background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #5AA72D,
        stop :   0.5 #B3E296, stop :   0.55 #B3E296, stop :   1.0 #f5f9ff);
        padding-top: 2px;
        padding-left: 3px;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>797</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="2" column="14">
      <widget class="QPushButton" name="Si">
       <property name="text">
        <string>Si</string>
       </property>
      </widget>
     </item>
     <item row="3" column="17">
      <widget class="QPushButton" name="Br">
       <property name="text">
        <string>Br</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QPushButton" name="Mn">
       <property name="text">
        <string>Mn</string>
       </property>
      </widget>
     </item>
     <item row="3" column="11">
      <widget class="QPushButton" name="Cu">
       <property name="text">
        <string>Cu</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QPushButton" name="Cr">
       <property name="text">
        <string>Cr</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="V">
       <property name="text">
        <string>V</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="Ti">
       <property name="text">
        <string>Ti</string>
       </property>
      </widget>
     </item>
     <item row="3" column="9">
      <widget class="QPushButton" name="Co">
       <property name="text">
        <string>Co</string>
       </property>
      </widget>
     </item>
     <item row="3" column="16">
      <widget class="QPushButton" name="Se">
       <property name="text">
        <string>Se</string>
       </property>
      </widget>
     </item>
     <item row="2" column="18">
      <widget class="QPushButton" name="Ar">
       <property name="text">
        <string>Ar</string>
       </property>
      </widget>
     </item>
     <item row="3" column="10">
      <widget class="QPushButton" name="Ni">
       <property name="text">
        <string>Ni</string>
       </property>
      </widget>
     </item>
     <item row="2" column="17">
      <widget class="QPushButton" name="Cl">
       <property name="text">
        <string>Cl</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="K">
       <property name="text">
        <string>K</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="Ca">
       <property name="text">
        <string>Ca</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="Sc">
       <property name="text">
        <string>Sc</string>
       </property>
      </widget>
     </item>
     <item row="3" column="12">
      <widget class="QPushButton" name="Zn">
       <property name="text">
        <string>Zn</string>
       </property>
      </widget>
     </item>
     <item row="3" column="8">
      <widget class="QPushButton" name="Fe">
       <property name="text">
        <string>Fe</string>
       </property>
      </widget>
     </item>
     <item row="3" column="13">
      <widget class="QPushButton" name="Ga">
       <property name="text">
        <string>Ga</string>
       </property>
      </widget>
     </item>
     <item row="3" column="14">
      <widget class="QPushButton" name="Ge">
       <property name="text">
        <string>Ge</string>
       </property>
      </widget>
     </item>
     <item row="3" column="15">
      <widget class="QPushButton" name="As">
       <property name="text">
        <string>As</string>
       </property>
      </widget>
     </item>
     <item row="2" column="13">
      <widget class="QPushButton" name="Al">
       <property name="text">
        <string>Al</string>
       </property>
      </widget>
     </item>
     <item row="2" column="16">
      <widget class="QPushButton" name="S">
       <property name="text">
        <string>S</string>
       </property>
      </widget>
     </item>
     <item row="1" column="18">
      <widget class="QPushButton" name="Ne">
       <property name="text">
        <string>Ne</string>
       </property>
      </widget>
     </item>
     <item row="1" column="17">
      <widget class="QPushButton" name="F">
       <property name="text">
        <string>F</string>
       </property>
      </widget>
     </item>
     <item row="1" column="16">
      <widget class="QPushButton" name="O">
       <property name="text">
        <string>O</string>
       </property>
      </widget>
     </item>
     <item row="1" column="15">
      <widget class="QPushButton" name="N">
       <property name="text">
        <string>N</string>
       </property>
      </widget>
     </item>
     <item row="1" column="14">
      <widget class="QPushButton" name="C">
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="1" column="13">
      <widget class="QPushButton" name="B">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="Be">
       <property name="text">
        <string>Be</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="Li">
       <property name="text">
        <string>Li</string>
       </property>
      </widget>
     </item>
     <item row="0" column="18">
      <widget class="QPushButton" name="He">
       <property name="text">
        <string>He</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="H">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>H</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="Y">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="Sr">
       <property name="text">
        <string>Sr</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="Rb">
       <property name="text">
        <string>Rb</string>
       </property>
      </widget>
     </item>
     <item row="3" column="18">
      <widget class="QPushButton" name="Kr">
       <property name="text">
        <string>Kr</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="Na">
       <property name="text">
        <string>Na</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="Mg">
       <property name="text">
        <string>Mg</string>
       </property>
      </widget>
     </item>
     <item row="4" column="7">
      <widget class="QPushButton" name="Tc">
       <property name="text">
        <string>Tc</string>
       </property>
      </widget>
     </item>
     <item row="4" column="8">
      <widget class="QPushButton" name="Ru">
       <property name="text">
        <string>Ru</string>
       </property>
      </widget>
     </item>
     <item row="4" column="9">
      <widget class="QPushButton" name="Rh">
       <property name="text">
        <string>Rh</string>
       </property>
      </widget>
     </item>
     <item row="4" column="10">
      <widget class="QPushButton" name="Pd">
       <property name="text">
        <string>Pd</string>
       </property>
      </widget>
     </item>
     <item row="4" column="11">
      <widget class="QPushButton" name="Ag">
       <property name="text">
        <string>Ag</string>
       </property>
      </widget>
     </item>
     <item row="4" column="12">
      <widget class="QPushButton" name="Cd">
       <property name="text">
        <string>Cd</string>
       </property>
      </widget>
     </item>
     <item row="4" column="13">
      <widget class="QPushButton" name="In">
       <property name="text">
        <string>In</string>
       </property>
      </widget>
     </item>
     <item row="4" column="14">
      <widget class="QPushButton" name="Sn">
       <property name="text">
        <string>Sn</string>
       </property>
      </widget>
     </item>
     <item row="4" column="15">
      <widget class="QPushButton" name="Sb">
       <property name="text">
        <string>Sb</string>
       </property>
      </widget>
     </item>
     <item row="4" column="16">
      <widget class="QPushButton" name="Te">
       <property name="text">
        <string>Te</string>
       </property>
      </widget>
     </item>
     <item row="4" column="17">
      <widget class="QPushButton" name="I">
       <property name="text">
        <string>I</string>
       </property>
      </widget>
     </item>
     <item row="4" column="18">
      <widget class="QPushButton" name="Xe">
       <property name="text">
        <string>Xe</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="Cs">
       <property name="text">
        <string>Cs</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="Ba">
       <property name="text">
        <string>Ba</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="Lanathanide_Series">
       <property name="text">
        <string>57-70</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="Lu">
       <property name="text">
        <string>Lu</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="Hf">
       <property name="text">
        <string>Hf</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QPushButton" name="Ta">
       <property name="text">
        <string>Ta</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QPushButton" name="W">
       <property name="text">
        <string>W</string>
       </property>
      </widget>
     </item>
     <item row="5" column="7">
      <widget class="QPushButton" name="Re">
       <property name="text">
        <string>Re</string>
       </property>
      </widget>
     </item>
     <item row="5" column="8">
      <widget class="QPushButton" name="Os">
       <property name="text">
        <string>Os</string>
       </property>
      </widget>
     </item>
     <item row="5" column="9">
      <widget class="QPushButton" name="Ir">
       <property name="text">
        <string>Ir</string>
       </property>
      </widget>
     </item>
     <item row="5" column="10">
      <widget class="QPushButton" name="Pt">
       <property name="text">
        <string>Pt</string>
       </property>
      </widget>
     </item>
     <item row="5" column="11">
      <widget class="QPushButton" name="Au">
       <property name="text">
        <string>Au</string>
       </property>
      </widget>
     </item>
     <item row="5" column="12">
      <widget class="QPushButton" name="Hg">
       <property name="text">
        <string>Hg</string>
       </property>
      </widget>
     </item>
     <item row="5" column="13">
      <widget class="QPushButton" name="Tl">
       <property name="text">
        <string>Tl</string>
       </property>
      </widget>
     </item>
     <item row="5" column="14">
      <widget class="QPushButton" name="Pb">
       <property name="text">
        <string>Pb</string>
       </property>
      </widget>
     </item>
     <item row="5" column="15">
      <widget class="QPushButton" name="Bi">
       <property name="text">
        <string>Bi</string>
       </property>
      </widget>
     </item>
     <item row="5" column="16">
      <widget class="QPushButton" name="Po">
       <property name="text">
        <string>Po</string>
       </property>
      </widget>
     </item>
     <item row="5" column="17">
      <widget class="QPushButton" name="At">
       <property name="text">
        <string>At</string>
       </property>
      </widget>
     </item>
     <item row="5" column="18">
      <widget class="QPushButton" name="Rn">
       <property name="text">
        <string>Rn</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="Fr">
       <property name="text">
        <string>Fr</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="Ra">
       <property name="text">
        <string>Ra</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="Actinide_Series">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>89-102</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="Lr">
       <property name="text">
        <string>Lr</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QPushButton" name="Rf">
       <property name="text">
        <string>Rf</string>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QPushButton" name="Db">
       <property name="text">
        <string>Db</string>
       </property>
      </widget>
     </item>
     <item row="6" column="6">
      <widget class="QPushButton" name="Sg">
       <property name="text">
        <string>Sg</string>
       </property>
      </widget>
     </item>
     <item row="6" column="7">
      <widget class="QPushButton" name="Bh">
       <property name="text">
        <string>Bh</string>
       </property>
      </widget>
     </item>
     <item row="6" column="8">
      <widget class="QPushButton" name="Hs">
       <property name="text">
        <string>Hs</string>
       </property>
      </widget>
     </item>
     <item row="6" column="9">
      <widget class="QPushButton" name="Mt">
       <property name="text">
        <string>Mt</string>
       </property>
      </widget>
     </item>
     <item row="6" column="10">
      <widget class="QPushButton" name="Uun">
       <property name="text">
        <string>Uun</string>
       </property>
      </widget>
     </item>
     <item row="6" column="11">
      <widget class="QPushButton" name="Uuu">
       <property name="text">
        <string>Uuu</string>
       </property>
      </widget>
     </item>
     <item row="6" column="12">
      <widget class="QPushButton" name="Uub">
       <property name="text">
        <string>Uub</string>
       </property>
      </widget>
     </item>
     <item row="6" column="14">
      <widget class="QPushButton" name="Uuq">
       <property name="text">
        <string>Uuq</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="Zr">
       <property name="text">
        <string>Zr</string>
       </property>
      </widget>
     </item>
     <item row="2" column="15">
      <widget class="QPushButton" name="P">
       <property name="text">
        <string>P</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="Nb">
       <property name="text">
        <string>Nb</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QPushButton" name="Mo">
       <property name="text">
        <string>Mo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>654</x>
      <y>402</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_firstElement">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_SecondElement">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>319</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Compound">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>369</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>691</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </widget>
   <widget class="QPushButton" name="fuseButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>320</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>FUSE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
